src/TurbulenceClosures/turbulence_closure_implementations/TKEBasedVerticalDiffusivities/tke_dissipation_vertical_diffusivity.jl

```diff
@@ -0,0 +1,356 @@
+                                                                       stability_functions,
+                                                                       minimum_length_scale,
+                                                                       maximum_tracer_diffusivity,
+                                                                       maximum_tke_diffusivity,
+                                                                       maximum_dissipation_diffusivity,
+                                                                       maximum_viscosity,
+                                                                       minimum_tke,
+                                                                       minimum_stratification_number_safety_factor,
+                                                                       negative_tke_damping_time_scale,
+                                                                       tke_dissipation_time_step)
+end
+
+TKEDissipationVerticalDiffusivity(FT::DataType; kw...) =
+    TKEDissipationVerticalDiffusivity(VerticallyImplicitTimeDiscretization(), FT; kw...)
+
+const TDVD{TD} = TKEDissipationVerticalDiffusivity{TD} where TD
+const TDVDArray{TD} = AbstractArray{<:TDVD{TD}} where TD
+const FlavorOfTD{TD} = Union{TDVD{TD}, TDVDArray{TD}} where TD
+
+@inline Base.eltype(::TKEDissipationVerticalDiffusivity{<:Any, <:Any, <:Any, <:Any, FT}) where FT = FT
+
+"""
+    TKEDissipationVerticalDiffusivity([time_discretization = VerticallyImplicitTimeDiscretization(),
+                                      FT = Float64;]
+                                      tke_dissipation_equations = TKEDissipationEquations(),
+                                      stability_functions = VariableStabilityFunctions(),
+                                      minimum_length_scale = StratifiedDisplacementScale(),
+                                      maximum_tracer_diffusivity = Inf,
+                                      maximum_tke_diffusivity = Inf,
+                                      maximum_dissipation_diffusivity = Inf,
+                                      maximum_viscosity = Inf,
+                                      minimum_tke = 1e-6,
+                                      minimum_stratification_number_safety_factor = 0.73,
+                                      negative_tke_damping_time_scale = 1minute,
+                                      tke_dissipation_time_step = nothing)
+
+Return the `TKEDissipationVerticalDiffusivity` turbulence closure for vertical mixing by
+small-scale ocean turbulence based on the prognostic evolution of subgrid
+Turbulent Kinetic Energy (TKE).
+
+!!! note "TKEDissipationVerticalDiffusivity"
+    `TKEDissipationVerticalDiffusivity` is a new turbulence closure diffusivity. The default
+    values for its free parameters are obtained from calibration against large eddy
+    simulations. For more details please refer to [Wagner23catke](@cite).
```

Comment:
We don't mention here that this is otherwise known as the "k-epsilon". I think we should. Perhaps a reference? Or at least mention that is inspired or a simplified version of...

Also, why do we claim that this is "a new turbulence closure diffusivity"? Perhaps you meant to say "unvalidated" or something?

Reply:
Maybe "newly implemented" or "recently implemented feature"? I agree that the wording seems like it is being claimed that we developed the parameterization.

Reply:
I deleted it

Reply:
Those words were leftover from CATKE (I copy pasted the whole file from the CATKE implementation)

Reply:
I put a few referencessin there and removed the warning about validation